[control-plane-manager] Add etcd backup (#9637)
deckhouse/deckhouse: modules/040-control-plane-manager/docs/FAQ.md

## etc backup and restore

### What is done automatically

> CronJob `kube-system/d8-etcd-backup` is automatically started at 00:00 UTC+0. The result is saved in `/var/lib/etcd/etcd-backup.snapshot` on a random master in the cluster.

Reviewer: ```suggestion
CronJob `kube-system/d8-etcd-backup` is automatically started at 00:00 UTC+0. The result is saved in `/var/lib/etcd/etcd-backup.snapshot` on a random control-plane node in the cluster.
```

> CronJob `kube-system/d8-etcd-backup` starts automatically at 00:00 UTC+0. The result is saved in `/var/lib/etcd/etcd-backup.snapshot` on a random control-plane node in the cluster.

Reviewer 2: ```suggestion
CronJob `kube-system/d8-etcd-backup` starts automatically at 00:00 UTC+0. The result is saved in `/var/lib/etcd/etcd-backup.snapshot` directory on a random control-plane node in the cluster.
```

> CronJob `kube-system/d8-etcd-backup-*` is automatically started at 00:00 UTC+0. The result is saved in `/var/lib/etcd/etcd-backup.snapshot` on all nodes with `control-plane` in the cluster.

Reviewer 2: ```suggestion
CronJob `kube-system/d8-etcd-backup-*` is automatically started at 00:00 UTC+0. The result is saved in `/var/lib/etcd/etcd-backup.snapshot` on all nodes with `control-plane` in the cluster (master nodes).
```

> CronJob `kube-system/d8-etcd-backup` starts automatically at 00:00 UTC+0. The result is saved in `/var/lib/etcd/etcd-backup.snapshot` directory on a random control-plane node in the cluster.

Reviewer 3: Not random. Result is saved on the master node where job performed.

Author: The job runs on a random master. More precisely on a less loaded master.
